src/controllers/controlpickermenu.cpp

```diff
@@ -354,46 +354,53 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
 
     // Library Controls
     QMenu* libraryMenu = addSubmenu(tr("Library"));
-    addPrefixedControl("[Playlist]", "ToggleSelectedSidebarItem",
-                       tr("Expand/Collapse View"),
-                       tr("Expand/collapse the selected view (library, playlist..)"),
+    addPrefixedControl("[Library]", "MoveUp",
+                       tr("Move up"),
+                       tr("Equivalent to pressing the UP key on the keyboard"),
                        m_libraryStr, libraryMenu);
-
-    addPrefixedControl("[Playlist]", "SelectPlaylist",
-                       tr("Switch Next/Previous View"),
-                       tr("Switch to the next or previous view (library, playlist..)"),
+    addPrefixedControl("[Library]", "MoveDown",
+                       tr("Move down"),
+                       tr("Equivalent to pressing the DOWN key on the keyboard"),
+                       m_libraryStr, libraryMenu);
+    addPrefixedControl("[Library]", "MoveVertical",
+                       tr("Move up/down"),
+                       tr("Move vertically in either direction using a knob, as if pressing UP/DOWN keys"),
+                       m_libraryStr, libraryMenu);
+    addPrefixedControl("[Library]", "ScrollUp",
+                       tr("Scroll Up"),
+                       tr("Equivalent to pressing the PAGE UP key on the keyboard"),
                        m_libraryStr, libraryMenu);
-    addPrefixedControl("[Playlist]", "SelectNextPlaylist",
```

Comment:
> and we can safely remove the old deprecated controls from the MIDI learn wizard in order to discourage their use?

Yes.  .. and there is no reason to delay this PR for future decisions. 

But lets try to find a conclusion for th new library layout as well (without delaying this) 

In a somehow designed n-Pane library, we have the issue that a put "TAB" / TAB_BACK" binding cycles though all controls. This is required to control the entire GUI without mouse, but it is not that straight forward for a default controller cursor mapping where we are focused on "Load A Track". 

TAB it self is handy on th other hand to directly switch between panes. This should be an ideal candiate for the DEON Shift+Back (Panes) button. 

If we are looking to a complex example: https://github.com/mixxxdj/mixxx/pull/991#issuecomment-268428023 , 
and a user wants to load a track from an old history using the keyboard, he has to do this: 
* UP/DOWN for history feature 
* Enter For enable the history list 
* TAB for history list
* UP/DOWN for selecting the right group of histories
* RIGHT for expand group 
* UP/DOWN for selecting and enable the desire history in the history track table. 
* TAB -> TAB -> TAB for going to the selected history track table 
* UP/DOWN for selecting the desired track. 

The TAB row becomes longer if we consider a button pane for AutoDJ and future features like Notepad or Lyrics or track suggestor. 

This leads to a following proposal for the above action using the controller's cursor keys:
* UP/DOWN for history feature 
* RIGHT for switching to history list and move the focus to it  
* UP/DOWN for selecting the right group of histories
* RIGHT for expand group 
* UP/DOWN for selecting the desire history
* RIGHT again for switching the history track table and move the focus to it 
* UP/DOWN for selecting the desired track. 

TAB / TAB_BACK could go to Shift + Right / Left
Shift UP/DOWN seams to be a good PgUp PgDown candidate. 

Adopted to the DENON controller it is 
* Turn Knob  for history feature 
* Push Knob for switching to history list and move the focus to it  
* Turn Knob for selecting the right group of histories
* Push Knob for expand group 
* Turn Knob for the desired history
* Push Knob for switching the history track table and move the focus to it 
* Turn Knob for selecting the desired track. 
 
 This looks very convenient to me. 

It does not map atomic to a keyboard key, but It reuses right, when it would be normally ignored by the widget. So we may consider to back port this behavior to the keyboard. 

What do you think?